change/@ni-nimble-components-a9107da1-a4c4-492b-9842-3d65680f0a8c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
```suggestion
  "type": "minor",
```

I think it should of type `minor` because we are introducing a new token.

Reply:
Updated